README.md

```diff
@@ -99,7 +99,7 @@ Reading package lists... Done
 
 ## Installation
 
-On OS X, you can install *The Fuck* via [Homebrew][homebrew]:
+On OS X or linux systems with linuxbrew, you can install *The Fuck* via [Homebrew][homebrew]:
```

Comment:
I'd rather like it to read something like:

```
On OS X, you can install *The Fuck* via [Homebrew][homebrew] (or via [Linuxbrew][linuxbrew] on Linux):
```

Reply:
Done